descriptors/ModuleDescriptor-template.json

```diff
@@ -549,7 +549,6 @@
             "inventory-storage.holdings.collection.get",
             "inventory-storage.holdings.item.post",
             "inventory-storage.loan-types.collection.get",
-            "inventory-storage.holdings-sources.collection.get",
```

Comment:
it is exists in line 547 https://github.com/folio-org/mod-orders/pull/914/files#diff-447a26c098aa9f236a852f39464c3149d97a871c75f5b9e8ee6e968bdebe319dR547

Reply:
deleted, godd catch